app/components/formToJson/formToJson.js

```diff
@@ -0,0 +1,13 @@
+function formToJson(){
+	//Convert formdata to json
+	$.fn.formToJson = function(){
```

Comment:
Rename to "toJson()". Check if 'this' is a form. If it is, continue the logic. If it is not, console an error saying that the method is just for forms.
Maybe in the future we can add some behaviour to be reuse in another kind of elements.